config/config.go

```diff
@@ -465,6 +465,7 @@ func NewConfig() (c *Config) {
 				},
 				Enabled: true,
 				IsCore:  false,
+				InClusterURL: "http://grafana.istio-system:3000",
```

Comment:
If we are sorting/alphabetizing all of these settings in here, then you should move this up a line. I'll approve in case we want to merge anyway.

Reply:
you need to gofmt this anyway (the format linter failed) - when you do it, just move it up a line.